packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/addconfigpkg.md

```diff
@@ -0,0 +1,6 @@
+Add the latest Okta OIDC package to your project:
+
+1. Choose **File** > **Add Packages**. The Swift Package Manager opens.
+1. Enter the URL for the package in the search field:
+`https://github.com/okta/okta-oidc-ios.git`
+1. Confirm that the Dependency Rule is set to the master branch and that your project is selected, then add the package.
```

Comment:
I think there should be an "and" after the last comma.

Reply:
fixed in my next PR